lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -80,9 +80,10 @@ export class PublicClientApplication implements IPublicClientApplication {
 
     /**
      * Initializer function to perform async startup tasks such as connecting to WAM extension
+     * @param correlationId {?string} correlation id
      */
-    async initialize(): Promise<void> {
-        return this.controller.initialize();
+    async initialize(correlationId?: string): Promise<void> {
```

Comment:
If we have to add this to public surface I would suggest making this an object in case we need to add additional input params/config in the future.

However, I would also suggest that maybe this doesn't need to be exposed yet. Can we start with simply generating a correlationId each time this is called?

Reply:
Also if we do need it passed in we should make sure the createPublicClientApplication APIs accept it as well

Reply:
Made it an object and added to the static `createPublicClientApplication` constructor.